libriscan/biblios/templates/biblios/components/layout/welcome_section.html

```diff
@@ -74,7 +74,251 @@ <h2 class="text-4xl font-bold mb-3">Welcome to LibriScan!</h2>
+                </svg>
+              </a>
+            {% endif %}
+          </div>
+          {% endif %}
+        {% else %}
+          <div class="alert">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" class="stroke-current shrink-0 w-6 h-6">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"></path>
+            </svg>
+            <span>No documents available. {% if user.is_staff or user.role == 'archivist' %}No documents have been created in your organization yet.{% else %}You haven't created any documents yet. Start transcribing to create your first document.{% endif %}</span>
+          </div>
+        {% endif %}
+      </div>
+    </div>
+
+    <!-- Tab Content 2: Pending Reviews -->
+    {% if user.is_staff or user.role == 'archivist' %}
+    <div id="tab2" class="tab-panel hidden">
+      <div id="pending-reviews">
+        {% if pending_reviews %}
+          <div class="overflow-x-auto">
+            <table class="table table-zebra w-full">
+              <thead>
+                <tr>
+                  <th class="text-base w-[15%]">Document ID</th>
+                  <th class="text-base w-[15%]">Organization</th>
+                  <th class="text-base w-[20%]">Collection</th>
+                  <th class="text-base w-[15%]">Series</th>
+                  <th class="text-base w-[15%]">Status</th>
+                  <th class="text-base w-[20%]">Actions</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for doc in pending_reviews %}
+                <tr class="hover">
+                  <td class="font-medium">{{ doc.identifier }}</td>
+                  <td>{{ doc.collection.owner.short_name }}</td>
+                  <td>{{ doc.collection.name }}</td>
+                  <td>{{ doc.series.name|default:"<span class='text-base-content/50'>Orphaned</span>"|safe }}</td>
+                  <td><span class="badge badge-warning">{{ doc.get_status_display }}</span></td>
+                  <td>
+                    <a href="{{ doc.get_absolute_url }}" class="btn btn-sm btn-warning">
+                      Review
+                      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-4 h-4">
+                        <path stroke-linecap="round" stroke-linejoin="round" d="M2.036 12.322a1.012 1.012 0 0 1 0-.639C3.423 7.51 7.36 4.5 12 4.5c4.638 0 8.573 3.007 9.963 7.178.07.207.07.431 0 .639C20.577 16.49 16.64 19.5 12 19.5c-4.638 0-8.573-3.007-9.963-7.178Z" />
+                        <path stroke-linecap="round" stroke-linejoin="round" d="M15 12a3 3 0 1 1-6 0 3 3 0 0 1 6 0Z" />
+                      </svg>
+                    </a>
+                  </td>
+                </tr>
+                {% endfor %}
+              </tbody>
+            </table>
+          </div>
```

Comment:
Now you can reuse the document_table like Tab1
```suggestion
          {% include 'biblios/components/tables/document_table.html' with documents=pending_reviews badge_class='badge-warning' action_text='Review' action_class='btn-warning' action_icon='eye_icon' %}
```